v21.1/update-data.md

```diff
@@ -62,20 +103,143 @@ if _, err := tx.Exec("UPDATE accounts SET balance = balance - $1 WHERE id = $2",
 ~~~ java
 // ds is an org.postgresql.ds.PGSimpleDataSource
 
-int transferAmount = 100;
-int fromID = 1;
+String ownerID = "bd70a3d7-0a3d-4000-8000-000000000025";
 
 try (Connection connection = ds.getConnection()) {
-    connection.createStatement().executeUpdate("UPDATE accounts SET balance = balance - "
-                                               + transferAmount + " where id = " + fromID);
+    connection.createStatement()
+            .executeUpdate("UPDATE vehicles SET status = 'unavailable' WHERE owner_id = '" + ownerID + "'");
 
 } catch (SQLException e) {
-    System.out.printf("sql state = [%s]\ncause = [%s]\nmessage = [%s]\n",
-                      e.getSQLState(), e.getCause(), e.getMessage());
+    System.out.printf("sql state = [%s]\ncause = [%s]\nmessage = [%s]\n", e.getSQLState(), e.getCause(),
+            e.getMessage());
+}
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="python">
+
+{% include copy-clipboard.html %}
+~~~ python
+# conn is a psycopg2 connection
+
+ownerID = 'bd70a3d7-0a3d-4000-8000-000000000025'
+
+with conn.cursor() as cur:
+    cur.execute(
+        "UPDATE vehicles SET status = 'unavailable' WHERE owner_id = %s", (ownerID,))
+    conn.commit()
```

Comment:
should the commit be outside of the `with` block? i'm actually not sure off the top of my head which is right, but this differs from the DELETE code sample